projects/1.16.1/assets/woot/woot/lang/zh_cn.json

```
@@ -0,0 +1,535 @@
+  "guide.woot.factory_blocks.sec_base.0": "一个有效的$(item)生物控制器$(0)可能会放置于这个方块上方。",
+  "guide.woot.factory_blocks.sec_base.name": "次级底座",
+  "guide.woot.machines.dyesqueezer.0": "$(item)染料液化机$(0)需求染料，或者可以制做成染料的物品然后将其转化成红，黄，蓝，白染料流体。它们可以合并成纯净染料精华，纯净染料精华可以在$(item)注塑机$(0)中消耗染料框架制造出任何染料板。",
+  "guide.woot.machines.dyesqueezer.1": "物品可以从任意面输入。纯净染料精华可以从任意面输出。$(item)染料液化机$(0)会自动输出到可接受的储罐中。",
+  "guide.woot.machines.dyesqueezer.name": "染料液化机",
+  "guide.woot.machines.enchsqueezer.0": "$(item)附魔液化机$(0)接受附魔物品（比如书，剑），提取出附魔并制造出液态魔咒。高等级附魔产生更多液体魔咒。",
+  "guide.woot.machines.enchsqueezer.1": "物品可以从任意面输入，液态魔咒可以从任意面输出。$(item)附魔液化机$(0)会自动输出到可接受的储罐中。",
+  "guide.woot.machines.enchsqueezer.2": "$(li) 附魔等级 I 500mb $(li) 附魔等级 II 1000mb $(li) 附魔等级 III 1500mb $(li) 附魔等级 IV 2000mb $(li) 附魔等级 V 2500mb $(li) 每高一级 500mb",
+  "guide.woot.machines.enchsqueezer.name": "附魔液化机",
+  "guide.woot.machines.fluidconvertor.0": "生物工厂需要萌生原液运行，主要由$(item)流体反应桶$(0)生产。这个机器消耗流体和触媒将其转化成萌生原液。不同的流体/触媒组合会产出不同数量的萌生原液。",
+  "guide.woot.machines.fluidconvertor.1": "物品可以从任意面输入。流体可以从上面，左面，前面输入，下面，右面，后面输出。",
+  "guide.woot.machines.fluidconvertor.name": "流体反应桶",
+  "guide.woot.machines.infuser.0": "$(item)注塑机$(0)将流体注入物品。它可以使用纯净染料精华来制造染料板。他也可以使用液体魔咒来制造工厂升级板和附魔书。",
+  "guide.woot.machines.infuser.1": "物品可以从上面，左面，前面输入，下面，右面，后面输出。流体可以从任意面输入。",
+  "guide.woot.machines.infuser.name": "注塑机",
+  "guide.woot.perks.crusher.0": "这个升级插件生产工业先锋的精华液于掉落物输出口旁的储罐。更高的等级会产出更多的流体。",
+  "guide.woot.perks.crusher.1": "$(li)等级 I - 100%%$(li)等级 II - 120%%$(li)等级 III - 140%%",
+  "guide.woot.perks.crusher.name": "压榨战斗",
+  "guide.woot.perks.efficiency.0": "这个升级插件降低工厂的萌生原液需求。更高等级更多降低的萌生原液需求。",
+  "guide.woot.perks.efficiency.1": "$(li)等级 I - 15%%减少$(li)等级 II - 25%%减少$(li)等级 III - 30%%减少",
+  "guide.woot.perks.efficiency.name": "振奋锅炉",
+  "guide.woot.perks.headless.0": "$(item)从“头”开始$(0)升级插件能够产生头颅掉落。更高的等级提供更高的掉落率",
+  "guide.woot.perks.headless.1": "$(li)等级 I - 25%%几率$(li)等级 II - 50%%几率$(li)等级 III - 80%%几率",
+  "guide.woot.perks.headless.name": "从“头”开始",
+  "guide.woot.perks.looting.0": "$(item)金银岛$(0)升级插件设定生物工厂使用的抢夺等级。抢夺等级从 I 到 III。",
+  "guide.woot.perks.looting.name": "金银岛",
+  "guide.woot.perks.mass.0": "$(item)横冲直撞$(0)升级插件增加工厂生成的生物数量。更高等级生成更多数量。如果工厂包含多种生物，将使用质量最小的生物。",
+  "guide.woot.perks.mass.1": "$(li)等级 I - 2只生物$(li)等级 II - 4只生物$(li)等级 III - 6只生物",
+  "guide.woot.perks.mass.name": "横冲直撞",
+  "guide.woot.perks.rate.0": "$(item)疯狂捕食$(0)升级插件减少生物生成所需的时间。更高的等级减少更多时间。如果工厂包含多种生物，将使用速率降低最小的生物。",
+  "guide.woot.perks.rate.1": "$(li)等级 I - 20%%减少$(li)等级 II - 50%%减少$(li)等级 III - 75%%减少",
+  "guide.woot.perks.rate.name": "疯狂捕食",
+  "guide.woot.perks.slaughter.0": "这个升级插件生产工业先锋粉红史莱姆液体和肉汤于掉落物输出口旁的储罐。更高的等级提高生产的流体量。",
+  "guide.woot.perks.slaughter.1": "$(li)等级 I - 100%%$(li)等级 II - 120%%$(li)等级 III - 140%%",
+  "guide.woot.perks.slaughter.name": "翻滚血肉玩偶",
+  "guide.woot.perks.tier_shard.0": "为了制造更高等级的生物工厂，你需要使用精华碎片来制造一些方块。精华碎片只有在安装了这个升级插件的工厂才能生产。不同等级的升级插件决定碎片生成的判定次数，工厂等级决定碎片掉落几率和不同碎片种类的掉落权重。",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "guide.woot.perks.tier_shard.0": "想要制造更高等级的生物工厂，你需要使用精华碎片来制造一些方块。只有在安装了这个升级插件之后，工厂才能生产精华碎片。不同等级的升级插件决定碎片生成的判定次数，工厂等级决定碎片掉落几率和不同碎片种类的掉落权重。",
```

想想平常是如何说话的：先说状语/条件，再说句子/结果:
> 昨天中午，我吃的是鱼。
> 只有按时吃饭，才不会得胃病。

一旦见到英语中的状语，记得把它倒装到前面，这比较符合汉语习惯。